tests/test_2dm.cpp

```diff
@@ -270,6 +271,98 @@ TEST( Mesh2DMTest, Basement3CellElevationTest )
   MDAL_CloseMesh( m );
 }
 
+
+
+TEST( Mesh2DMTest, SaveMeshToFile )
+{
+  //open a mesh
+  std::string pathSource = test_file( "/2dm/quad_and_triangle.2dm" );
+  MeshH meshToSave = MDAL_LoadMesh( pathSource.c_str() );
+
+  //save the mesh
+  std::string fileNameToSave = tmp_file( "/quad_and_triangle_saveTest.2dm" );
+  MDAL_SaveMesh( meshToSave, fileNameToSave.c_str() );
+
+  MDAL_CloseMesh( meshToSave );
+
+  //open the saved mesh and test it (same as QuadAndTriangleFile test)
```

Comment:
do not copy paste (DRY) ... either create common function to test both files, or just compare quad_and_triangle.2dm and quad_and_triangle_saveTest.2dm as text files?

Reply:
done